library/src/main/java/com/getstream/sdk/chat/view/messageinput/MessageInputController.kt

```diff
@@ -152,9 +169,20 @@ internal class MessageInputController(
     }
 
     internal fun onFileCaptured(file: File) {
+        attachmentsController.setSelectedAttachmentAdapter(null, true)
         attachmentsController.selectAttachment(AttachmentMetaData(file), true)
     }
 
+    internal fun onFilesSelected(uriList: List<Uri>) {
+        GlobalScope.launch(dispatchersProvider.mainDispatcher) {
```

Comment:
For me it's logic related to work with attachments. That's why this should be moved to AttachmentsController